src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +172,23 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::clone(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    if (info.Length() > 0) {
+        reportError(env, "clone() does not accept any arguments.");
+        return env.Undefined();
+    }
+    return ModelWrap::wrap(env, _model->clone());
```

Comment:
Please check how the approach with `std::vector<std::string> allowed_signatures;` can be applied here
https://github.com/openvinotoolkit/openvino/blob/771489d42d1d04194b8e8278bc4b373fe307b6fd/src/bindings/js/node/src/core_wrap.cpp#L63-L101

Reply:
Need a clarification here @almilosz 
Does the use of the allowed_signatures vector here function as a validator on the signature of the function that the JS API calls internally? If yes, I would just need to use the following inside the try catch: 

`if (ov::js::validate(info, allowed_signatures)) return cpp_to_js(env, model->clone())`

Reply:
I have made the changes as requested after studying the allowed_signatures pattern. 
Please check if my changes are in line with the expectation @almilosz 
Thanks.

Reply:
Since I didn't see any comment regarding the use of allowed_signatures, I assume my implementation is ok and am resolving this thread. Thanks